fe/fe-core/src/main/java/org/apache/doris/policy/PolicyMgr.java

```diff
@@ -179,12 +180,16 @@ public void dropPolicy(DropPolicyLog dropPolicyLog, boolean ifExists) throws Ddl
                 for (Table table : tables) {
                     if (table instanceof OlapTable) {
                         OlapTable olapTable = (OlapTable) table;
+                        String tableName = table.getName();
+                        if (table.isTemporary()) {
+                            tableName = Util.getTempTableDisplayName(tableName);
+                        }
```

Comment:
why not Util provide add new function to process Table object and return right name

Reply:
coz sometimes only have tablename with string type. refactor this code